server/controllers/content.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable sort-keys */
+/* eslint-disable consistent-return */
+const HttpStatus = require('http-status-codes');
+const Content = require("../models/Content");
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND } = HttpStatus;
+
+// Create and Save a new Content
+exports.create = async(req, res) => {
+    try {
+        const { title, resourceType, tags, url, subjectId } = req.body;
+        // Validate request
+        if (!title) {
+            return res.status(BAD_REQUEST)
+                .send({
+                    status: "fail",
+                    data: { "title": "Content title is required" },
+                });
+        }
+
+        // Create a Content
+        const content = new Content({
+            title,
+            resourceType,
+            tags,
+            url,
+            subjectId,
+        });
+
+        // Save Content in the database
+        const data = await content.save();
+        res.status(CREATED)
+            .send({
+                status: "success",
+                data: { "content": data },
+            });
+    } catch (err) {
+        res.status(INTERNAL_SERVER_ERROR)
+            .send({
+                status: "error",
+                message: HttpStatus.getStatusText(INTERNAL_SERVER_ERROR) ||
+                    "Some error occurred while creating the Content.",
+            });
+    };
+};
+
+// Retrieve and return all content from the database.
+exports.findAll = async(req, res) => {
+    try {
+        const data = await Content.find();
+        res.status(OK)
+            .send({
+                status: "success",
+                data: { "contents": data },
+            });
+
+    } catch (err) {
+        res.status(INTERNAL_SERVER_ERROR)
+            .send({
+                status: "error",
+                message: HttpStatus.getStatusText(INTERNAL_SERVER_ERROR) ||
+                    "Some error occurred while retrieving contents.",
+            });
+    };
+};
+
+// Find a single content with a contentId
+exports.findOne = async(req, res) => {
+    const { contentId } = req.params;
+    try {
+        const data = await Content.findById(contentId);
+        if (!data) {
+            return res.status(NOT_FOUND)
+                .send({
+                    status: "fail",
+                    data: { "id": `Content not found with id ${contentId}` },
+                });
+        }
+        res.status(OK)
+            .send({
+                status: "success",
+                data: { "content": data },
+            });
```

Comment:
just send
```
{
     status: "success",
     data,
}
```

Reply:
The JS End standard requires that you return JSON this way 
"data": {"entity" : the content of the instance of that entity}
![image](https://user-images.githubusercontent.com/2178492/57218621-a8b6df00-6fed-11e9-9a01-6bda1d91410a.png)